projects/fleet-management-grid/src/fleet-management-grid.ts

```diff
@@ -1,81 +1,1048 @@
+
+    <div class="overlay-backdrop" id="overlayBackdrop"></div>
+    <div class="overlay-wrapper" id="locationOverlay">
+      <div>
+        <igc-card elevated class="overlay overlay-location">
+          <igc-card-header class="overlay-location-header" vertical="true">
+            <div class="overlay-header-content">
+              <igc-avatar class="overlay-avatar" shape="circle" src="${this.vehicleDetails.vehiclePhoto}"></igc-avatar>
+              <h6 class="overlay-title">${this.vehicleDetails.make} ${this.vehicleDetails.model}</h6>
+              <span class="overlay-text">Mileage: ${this.vehicleDetails.mileage}</span>
+            </div>            
+          </igc-card-header>
+          <igc-card-content class="overlay-location-content">
+            <igc-geographic-map id="map" width="360px" height="190px"
+              zoomable="false"
+              draggable="false"
+              .dataSource="${this.vehicleDetails.markerLocations}"
+              latitude-member-path="latitude"
+              longitude-member-path="longitude">
+            </igc-geographic-map>
+          </igc-card-content>
+          <igc-card-actions class="overlay-location-actions">
+            <igc-button variant="flat" @click=${() => (this.closeLocationOverlay())}>Close</igc-button>
+          </igc-card-actions>
+        </igc-card>
+      </div>   
+    </div>  
+    
+    
+    <div class="overlay-wrapper" id="driverOverlay">
+      <igc-card elevated class="overlay overlay-driver">
+        <igc-card-header class="overlay-driver-header" vertical="true">
+          <div class="overlay-header-content">
+            <igc-avatar class="overlay-avatar" shape="circle" src="${this.driverDetails.photo}"></igc-avatar>
+            <h6 class="overlay-title">${this.driverDetails.name}</h6>
+          </div>
+        </igc-card-header>
+        <igc-card-content class="overlay-driver-content">
+          <div class="driver-block-container">
+            <div class="driver-category-container">
+              ${DRIVER_CATEGORIES.driverCategories.map(category => html`
+                <div class="detail-item">
+                  <span class="detail-category">${category.label}:</span>
+                  <igc-divider></igc-divider>
+                </div>
+              `)}
+            </div>
+            <div class="driver-content-container">
+              ${DRIVER_CATEGORIES.driverCategories.map(category => html`
+                <div class="detail-item">
+                  <span class="detail-value">${this.driverDetails[category.key]}</span>
+                  <igc-divider></igc-divider>
+                </div>
+              `)}
+            </div>
+          </div>
+        </igc-card-content>
+        <igc-card-actions class="overlay-driver-actions">
+          <igc-button variant="flat" @click=${() => (this.closeDriverOverlay())}>Close</igc-button>
+        </igc-card-actions>
+      </igc-card>
+    </div> 
+    `;
   }
 
 
   static styles = css`
```

Comment:
You should probably move those styles in another file because the file is getting too big . You have two approaches:
1. Just move this into a separate file like [this](https://github.com/IgniteUI/grid-demos-wc/pull/9/files#diff-4fc605ebb02740088bff90ee1acf44ee239903bbb58ae9bfad32de2ad935b863R1)
2. Or use scss file and import it like [this](https://github.com/IgniteUI/grid-demos-wc/pull/5/files#diff-56736ab07a823711ba715dacdbb7802e99b2ed9e21b4382d48bfea8bb881dbfeR1). Keep in mind that you must install `sass` so that vite has a way to preprocess the scss file